implementations/rust/router/src/lib.rs

```diff
@@ -1,80 +1,101 @@
-use ockam_common::commands::ockam_commands::*;
-use ockam_message::*;
+#![allow(unused)]
+pub mod router {
```

Comment:
Same comment as above re: `pub mod router` namespace wrapping -- since this is in `router/src/lib.rs` it's already in the `router` module.